src/Pages/archivePage/archive.js

```diff
@@ -1,27 +1,44 @@
 import React from 'react';
 import './archive.css';
 import { useArchiveNote } from '../../context/archive-note-context';
+import { useToastContext } from '../../context/toastContext';
 
 
 export default function Archive() {
   const archiveNotes = useArchiveNote();
+  const setArchiveNotes = useArchiveNote();
```

Comment:
Suggestion: You can import archiveNotes, and setArchiveNotes on same line using {}.